### ISSUE TITLE
Make sure RepositoryVersionValidatorsManagementController is following the How-To-Write-Controllers guide

### ISSUE THREAD

**Issue by steve-todorov** [strongbox/strongbox#609]
Currently the [RepositoryVersionValidatorsManagementController](https://github.com/strongbox/strongbox/blob/master/strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/RepositoryVersionValidatorsManagementController.java) is not fully following the [How-To-Write-Controllers](https://github.com/strongbox/strongbox/wiki/How-To-Implement-Spring-Controllers) guide. This needs to be fixed.

**Comment by ptirador** [comment]
I'm taking this one.

**Comment by steve-todorov** [comment]
@carlspring is this still on hold? :)

**Comment by carlspring** [comment]
@steve-todorov : I am in the middle of a big refactoring around the version validators, which are now a sub type of the newly introduced concept of artifact coordinate validators. The refactoring is almost ready, but it still needs some more work. There's no point in working on this until I'm done.

**Comment by ptirador** [comment]
Ok, then. I'll wait for it.

**Comment by carlspring** [comment]
This issue depends on #612 .

**Comment by carlspring** [comment]
@ptirador : This should now be good to go

**Comment by carlspring** [comment]
Implemented by @ptirador.
